index.js

```diff
@@ -91,7 +92,7 @@ module.exports = function (archive, target, opts, cb) {
       var rs = fs.createReadStream(file)
       var ws = archive.createWriteStream(hyperPath, {indexing: opts.indexing})
       entry = entries[hyperPath] = entry || {}
```

Comment:
Can we do `entry = opts.resume ? archive.stat(hyperPath) : null` here instead of using the history below?